config/default-example.yaml

```diff
@@ -15,9 +15,15 @@ authentication:
 dataSources:
   dataSources: ['oracledb']
   oracledb:
-    connectString: ${DB_URL}
-    user: ${DB_USER}
-    password: ${DB_PASSWD}
     poolMin: 4
     poolMax: 4
     poolIncrement: 0
+    oracleSources: ['bannerRead', 'odsRead']
+    bannerRead:
```

Comment:
As we discussed, I think we can just use `banner` and `ods` since banner read and banner write will be the same instance in v2.